src/Agent.Plugins/TeeCliManager.cs

```diff
@@ -231,10 +231,10 @@ public async Task UndoAsync(string localPath)
             await RunCommandAsync(FormatFlags.OmitCollectionUrl, "undo", "-recursive", localPath);
         }
 
-        public async Task UnshelveAsync(string shelveset)
+        public async Task UnshelveAsync(string shelveset, bool failOnNonZeroExitCode = false)
```

Comment:
Why is this one false by default but the other one above is true by default? I'd expect both to be true to avoid unintended behavior change